docs/nb/Exploring_Emax.ipynb

```diff
@@ -0,0 +1,641 @@
+{
```

Comment:
So I have not generally worked with ipython notebooks. From seeing the python code I can't make much of this. Is the ideas that these are demonstrative notebooks that you have developed during your working? I can see these as being worthwhile. But we must ensure that these are kept up-to-date so we don't end up with too much stuff on "main" that does not work.

Reply:
These are only for simple development and testing.

They are *not* part of the code base.  And therefore
need not be kept up to date.